test/integration/targets/vmware_content_library/content_lib_crud_ops.yml

```diff
@@ -0,0 +1,73 @@
+# Test code for the vmware_content_library CRUD Operations.
+# Copyright: (c) 2019, Pavan Bidkar <pbidkar@vmware.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- when: vcsim is not defined
+  block:
```

Comment:
Could you please start with an unconditional reset of the state of the resource. This to ensure the test is idempotent.

Reply:
I will make this change